scvi/model/_destvi.py

```diff
@@ -96,7 +98,9 @@ def from_rna_model(
         cls,
         st_adata: AnnData,
         sc_model: CondSCVI,
-        vamp_prior_p: int = 50,
+        vamp_prior_p: int = 500,
+        vamp_prior_resolution: float = 10.0,
+        l1_sparsity: float = 60.0,
```

Comment:
I feel that this prior default should be zero, I mostly didn't use it in the paper. Did you have some test supporting a diff value?

Reply:
No, I agree with setting it zero. It might be beneficial in edge cases, but also in those cases it makes sense to first run without it. Everything above 120 breaks performance (Tregs are predicted to be not present at all in lymph node).